src/Rules/Actions.hs

```diff
@@ -1,12 +1,14 @@
 module Rules.Actions (
     build, buildWithCmdOptions, buildWithResources, copyFile, fixFile, moveFile,
-    removeFile, copyDirectory, createDirectory, moveDirectory, removeDirectory,
-    applyPatch, runBuilder, makeExecutable, renderProgram, renderLibrary
+    removeFile, copyDirectory, copyDirectoryContent, createDirectory,
+    moveDirectory, removeDirectory, applyPatch, runBuilder, runBuilderWith,
+    makeExecutable, renderProgram, renderLibrary
     ) where
 
 import qualified System.Directory       as IO
 import qualified System.IO              as IO
 import qualified Control.Exception.Base as IO
+import qualified System.Directory.Extra as X
```

Comment:
No need to import System.Directory.Extra and System.Directory since the former reexports the latter.